azext_edge/edge/util/az_client.py

```diff
@@ -4,7 +4,8 @@
 # Licensed under the MIT License. See License file in the project root for license information.
 # ----------------------------------------------------------------------------------------------
 
-from collections.abc import MutableMapping
+# this is a false positive in pylint 3.0.3 for python 3.13
+from collections.abc import MutableMapping  # pylint: disable=import-error
```

Comment:
are you going to follow up on this in a next iteration

Reply:
yes - when we increase the version of pylint